### PR TITLE
Integrate BERT into Hedwig

### PR DESCRIPTION
**Pull request by achyudh**
### Changes:
* Fix package imports

* Update README.md

* Fix bug due to TAR/AR attribute check

* Add BERT models

* Add BERT tokenizer

* Return logits from the model.py

* Remove unused classes in models/bert

* Return logits from the model.py (#12)

* Remove unused classes in models/bert (#13)

* Add initial main file

* Add args for BERT

* Add partial support for BERT

* Initialize training and optimization

* Draft the structure of Trainers for BERT

* Remove duplicate tokenizer

* Add utils

* Move optimization to utils

* Add more structure for trainer

* Refactor the trainer (#15)

* Refactor the trainer

* Add more edits

* Add support for our datasets

* Add evaluator

* Split data4bert module into multiple processors

* Refactor BERT tokenizer

* Integrate BERT into Castor framework (#17)

* Remove unused classes in models/bert

* Split data4bert module into multiple processors

* Refactor BERT tokenizer

* Add multilabel support in BertTrainer

* Add multilabel support in BertEvaluator

* Add get_test_samples method in dataset processors

* Fix args.py for BERT

* Add support for Reuters, IMDB datasets for BERT

### CODE REVIEW

**Comment by Ashutosh-Adhikari** [comment]
Why would we create a duplicate PR? Or is there something that I am missing?

**Comment by achyudh** [comment]
I cannot resolve conflicts and push directly to karkaroff. It has to be on a fork I have write access to.

**Comment by Ashutosh-Adhikari** [comment]
Don't worry, I have resolved the conflicts. Please close this duplitcate PR. Thanks

**Comment by achyudh** [comment]
Hmm, I don't see any changes in your original pull request yet

**Comment by Ashutosh-Adhikari** [comment]
The conflicts cant be resolved due to karkaroff's merge access rule and @lintool 's commit. I am approving this and closing #10 .

**Review by Ashutosh-Adhikari — APPROVED** [review]
LGTM.
